Tiltfile

```diff
@@ -123,3 +129,43 @@ local_resource(
     serve_cmd=searcher_command,
     resource_deps=["deploy-contracts", "auction-server", "create-configs"],
 )
+
+
+# Solana resources
+local_resource(
+    "build-programs",
+    "cargo build-sbf",
+    dir="express_relay",
+)
+
+local_resource(
+    "solana-localnet",
+    serve_cmd="solana-test-validator $(./test-validator-params.sh)",
+    serve_dir="express_relay",
+    # check readiness by sending a health GET query to the RPC url
+    readiness_probe=probe(
+        period_secs=10,
+        http_get = http_get_action(port=int(rpc_port_solana), host="localhost", scheme="http", path="/health")
+    ),
+    resource_deps=["build-programs"],
+)
+
+local_resource(
+    "airdrop",
```

Comment:
I would call this resource and script `setup-accounts`. airdrop is a bit misleading since it actually checks and creates the keypairs if they don't exist

Reply:
another way to do this is to just create the keypairs and then start the `solana-localnet` with these accounts already topped up. This is possible via the `--account` flag and by providing such a json:
```
{
  "pubkey": "8WpVYzy6XZQy7HiRdRD4kNhGx7yr8K55QumUG2BVTyeA",
  "account": {
    "lamports": 1000000000,
    "data": [
      "",
      "base64"
    ],
    "owner": "11111111111111111111111111111111",
    "executable": false,
    "rentEpoch": 1,
    "space": 0
  }
}
```

I noticed it takes more than 30 seconds to have the airdrops so it may be worth it.